index.html

```diff
@@ -177,7 +177,7 @@ <h2 class="card-title">
             </article>
             <article class="card-work">
                 <div class="card-image">
-                    <img src="">
+                    <img src="./img/man-with-backpack-exploring-nature.jpg" alt="man exploring the nature">
```

Comment:
Well done in adding alt tags to your image tags. This will made it easier for people with low bandwidth to browse your page.